.circleci/config.yml

```diff
@@ -26,6 +26,7 @@ jobs:
             virtualenv .venv
             source .venv/bin/activate
             pip install --require-hashes -r test-requirements.txt
+            sudo apt install lsof
```

Comment:
`lsof` is available on Mac and under Linux, and therefore seemed like the best fit to perform a cross-platform test for exclusive lock access. It is, however, not installed in this particular CircleCI container by default.